### PR TITLE
Bah 4192 Part 1 and 2 Instructions

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
First two pages of 4192 Instructional flow.

## Testing done
No testing.

## Screenshots
screen shot 2019-01-16 at 9 53 08 am
screen shot 2019-01-16 at 9 51 29 am

## Acceptance criteria
- [ ] The content on the page titled Completing VA Form 21-4192 matches the 4192 Content Document (per link in Supporting Artifacts above/screenshot below)

- [ ]  the image displayed on this page is 5.2.1-image (see link in Supporting Artifacts)
The content on the page titled Fill in employer details matches the 4192 Content Document (per link in Supporting Artifacts above/screenshot below)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/4192/index.js [hunk 1]
```diff
@@ -25,6 +27,16 @@ export default function() {
...
+        path: '4192-instructions-part-1',
+        uiSchema: instructionalPart1.uiSchema,
+        schema: instructionalPart1.schema,
+      },
+      instructionalPart2: {
+        path: '4192-instructions-part-2',
+        uiSchema: instructionalPart2.uiSchema,
+        schema: instructionalPart2.schema,
+      },
       // Form Tutorial (multiple pages)
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/config/4192/index.js]: move this comment above your pages please

File: src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const instructionalPart2Description = (
+  <div>
+    <h3>Fill in employer details</h3>
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx]: other pages are using `<h4>`, might have to change other pages if `<h3>` is what we are supposed to be using

File: src/applications/disability-benefits/all-claims/pages/medicalCare.jsx [hunk 3]
```diff
@@ -19,13 +19,16 @@ export const uiSchema = {
...
+      'Please tell us if you were hospitalized under a doctor‘s care for the service-connected disabilities',
     'view:doctorsCare': {
       'ui:title':
-        'I‘ve been under a doctor‘s care in the past 12 months for these disabilities.',
+        'Yes, I‘ve been under a doctor‘s care in the past 12 months for these disabilities.',
     },
     'view:hospitalized': {
       'ui:title':
-        'I‘ve spent time in a hospital in the past 12 months for these disabilities.',
+        'Yes, I‘ve been hospitalized im the past 12 months for these disabilities.',
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/pages/medicalCare.jsx]: "im" instead of "in"

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by imeecuison** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/disability-benefits/all-claims/content/instructionalPart1.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const instructionalPart1Description = (
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/content/instructionalPart1.jsx]: Should this be a default export, if it's the only thing you expect to be in this file?

File: src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const instructionalPart2Description = (
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx]: Same question as above, should it be a default export?

**Review by jbalboni — APPROVED** [review]
A couple minor comments, LGTM otherwise.

File: src/applications/disability-benefits/all-claims/content/instructionalPart1.jsx (hunk 4 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/instructionalPart1.jsx]: changed. Thank you!

File: src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx (hunk 5 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/instructionalPart2.jsx]: changed. Thank you!

**Pull request merged**